### PR TITLE
[nri-statsd] Add DaemonSet option

### PR DESCRIPTION
**Pull request by thilinapiy**
#### Is this a new chart
No

#### What this PR does / why we need it:
This PR is adding an option to run StatsD as a `DaemonSet`

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: charts/nri-statsd/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 2.0.0
 description: A Helm chart to deploy the New Relic Statsd integration
 name: nri-statsd
-version: 1.0.0
+version: 1.0.2
```
Comment (varas) [inline charts/nri-statsd/Chart.yaml:5]: 🎗️  depends on https://github.com/newrelic/helm-charts/pull/372/files#diff-28e77334f2e9a2a31350c1e5e7f801593623f59a055eb4276bcf7c3a4c66b729R5

**Review by varas — APPROVED** [review]

File: charts/nri-statsd/README.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ This chart will deploy the New Relic's StatsD Integration. Keep in mind that due
...
 | `image.pullSecrets`                                        | Image pull secrets.                                                                                                                                                                                | `nil`                                  |
 | `resources`                                                | A yaml defining the resources for the events-router container.                                                                                                                                     | {}                                     |
+| `daemonset.create`                                         | If true, StatsD will be deployed as a daemonset insted of a deployment                                                                                                                             | `false`                                |
```
Comment (varas) [inline charts/nri-statsd/README.md:27]: typo here `instead`

File: charts/nri-statsd/Chart.yaml (hunk 1 above)
Comment (thilinapiy) [reply charts/nri-statsd/Chart.yaml]: Update the version.

File: charts/nri-statsd/README.md (hunk 2 above)
Comment (thilinapiy) [reply charts/nri-statsd/README.md]: fixed the typo.

**Review by varas — APPROVED** [review]

File: charts/nri-statsd/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 2.0.0
 description: A Helm chart to deploy the New Relic Statsd integration
 name: nri-statsd
-version: 1.0.0
+version: 1.0.3
```
Comment (varas) [inline charts/nri-statsd/Chart.yaml:5]: wait, this should go after https://github.com/newrelic/helm-charts/pull/372/files#diff-28e77334f2e9a2a31350c1e5e7f801593623f59a055eb4276bcf7c3a4c66b729R5 so it'd be 1.0.2

Comment (thilinapiy) [reply charts/nri-statsd/Chart.yaml]: set to 1.0.2.

**Comment by varas** [comment]
Hi @thilinapiy , released at https://github.com/newrelic/helm-charts/releases/tag/nri-statsd-1.0.2

**Pull request merged**